src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -333,7 +337,7 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configMempoolP2p = defaultEnableConfig defaultMempoolP2pConfig
     , _configBlockGasLimit = 150000
     , _configMinGasPrice = 1e-8
-    , _configPactQueueSize = 2000
+    , _configPactQueueSize = 10000
```

Comment:
What is the motivation for this. For all chains that would 20*10000 pact requests. Also if each pact requests takes a few ms (as typical), 10000 items would be a backlog of something like 10-20seconds at least.